server/api/video.api.js

```diff
@@ -17,22 +17,35 @@ const headersValidation = Joi.object({
   authorization: Joi.string().required(),
 }).options({ allowUnknown: true });
 
-module.exports = function(server, mongoose, logger) {
+module.exports = function (server, mongoose, logger) {
   // Update Video Segments Endpoint
-  (function() {
+  (function () {
     const Log = logger.bind(Chalk.magenta('Update Video Segments'));
 
     Log.note('Generating Update Video Segments Endpoint');
 
-    const updateVideoSegmentsHandler = async function(request, h) {
+    const updateVideoSegmentsHandler = async function (request, h) {
       try {
         const Segment = mongoose.model('segment');
 
         const { videoId, segments } = request.payload;
-        const video = (await RestHapi.list({
-          model: 'video',
-          query: { ytId: videoId, $embed: ['segments.tags'] },
-        })).docs[0];
+
+        //Log.debug("Segments length:", segments.length);
+        //Log.debug("Segments before:", segments, segments[0].tags);
```

Comment:
I'm sure I'm also guilty of this, but I think its a good practice to not commit/push commented code unless necessary. If it is necessary then usually a comment explaining why is appropriate.